WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -133,27 +133,28 @@ private void init() {
         mProgressLoadMore = (ProgressBar) findViewById(R.id.progress_loading);
         mProgressLoadMore.setVisibility(View.GONE);
 
-        mSwipeToRefreshHelper = new SwipeToRefreshHelper(getContext(),
-                (CustomSwipeRefreshLayout) findViewById(R.id.ptr_layout),
-                new SwipeToRefreshHelper.RefreshListener() {
-                    @Override
-                    public void onRefreshStarted() {
-                        post(new Runnable() {
-                            @Override
-                            public void run() {
-                                if (!NetworkUtils.checkConnection(getContext())) {
-                                    mSwipeToRefreshHelper.setRefreshing(false);
-                                    updateEmptyView(EmptyViewMessageType.NETWORK_ERROR);
-                                    return;
-                                }
-                                if (mFilterListener != null){
-                                    mFilterListener.onLoadData();
-                                }
+        mSwipeToRefreshHelper = new SwipeToRefreshHelper(
+            (CustomSwipeRefreshLayout) findViewById(R.id.ptr_layout),
+            new SwipeToRefreshHelper.RefreshListener() {
+                @Override
+                public void onRefreshStarted() {
+                    post(new Runnable() {
+                        @Override
+                        public void run() {
+                            if (!NetworkUtils.checkConnection(getContext())) {
+                                mSwipeToRefreshHelper.setRefreshing(false);
+                                updateEmptyView(EmptyViewMessageType.NETWORK_ERROR);
+                                return;
                             }
-                        });
-                    }
-                });
-
+                            if (mFilterListener != null){
+                                mFilterListener.onLoadData();
+                            }
+                        }
+                    });
+                }
+            },
+            R.color.color_primary, R.color.color_accent
```

Comment:
It seems that setting the colors is becoming a bit repetitive so, how about defining a helper method in them main WordPress module somewhere (say, in the util package) to return a new SwipeToRefreshHelper with the colors set?

Reply:
Good idea.  I added a helper helper with `WPSwipeToRefreshHelper` in 37f7e5de394b069859131c42c511562e9d8465d2 and updated the usages in 3c5946c8a782f09248d060f2828b48d6e55b0187.